nest/script/UpgradeNestContracts.s.sol

```diff
@@ -1,30 +1,57 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.25;
 
+import { AggregateToken } from "../src/AggregateToken.sol";
+import { AggregateTokenProxy } from "../src/proxy/AggregateTokenProxy.sol";
 import { UUPSUpgradeable } from "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
 import { Script } from "forge-std/Script.sol";
-import { Test } from "forge-std/Test.sol";
 import { console2 } from "forge-std/console2.sol";
 
-import { AggregateToken } from "../src/AggregateToken.sol";
-import { AggregateTokenProxy } from "../src/proxy/AggregateTokenProxy.sol";
-
-contract UpgradeNestContracts is Script, Test {
+contract UpgradeNestContracts is Script {
 
     address private constant NEST_ADMIN_ADDRESS = 0xb015762405De8fD24d29A6e0799c12e0Ea81c1Ff;
+
     UUPSUpgradeable private constant AGGREGATE_TOKEN_PROXY =
         UUPSUpgradeable(payable(0x659619AEdf381c3739B0375082C2d61eC1fD8835));
 
-    function test() public { }
```

Comment:
I think we should keep the `test()` so it doesn't show up in `forge coverage` as untested?